### PR TITLE
San 295 5 f visuals

### PR DESCRIPTION
**Pull request by psmatsinhe**

### CODE REVIEW

File: packages/cdr/docker/jsreport/scripts/report-5f/content.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                }
+              ]
+            : [])
+        ]
+      }
+    },
+    aggs: {
+      suppression: {
+        terms: {
+          field: 'viralSuppression.currentStatus'
```
[the remark this answers is not in the document]
Comment (psmatsinhe) [reply packages/cdr/docker/jsreport/scripts/report-5f/content.js]: Done @rcrichton

File: packages/cdr/docker/jsreport/scripts/report-5f/content.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+          {
+            exists: {
+              field: 'hivDiagnosis.hivPosDate'
+            }
+          },
+          {
+            exists: {
+              field: 'artInitiation.dateInitiated'
+            }
+          },
```
Comment (rcrichton) [inline packages/cdr/docker/jsreport/scripts/report-5f/content.js:32]: This is redundant as you already are using this field in the range query above.

File: packages/cdr/docker/jsreport/report-5f/helpers.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+function serialize (data) {
+    return JSON.stringify(data)
+  }
+  
+  function formatNumber (num) {
+    return (num || 0).toFixed(1)
+  }
```
Comment (rcrichton) [inline packages/cdr/docker/jsreport/report-5f/helpers.js:7]: Minor: the indentation here is wrong.

**Review by rcrichton — COMMENTED** [review]
@psmatsinhe some more minor comments for you. I'm still going through this but I'm having a little trouble testing atm. Trying to figure out what is wrong in my setup.

File: packages/cdr/docker/jsreport/scripts/report-5f/content.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+  const { from, to, state, district, city, facilityCode } = req.data.params
+
+  const esQuery = {
+    size: 0,
+    query: {
+      bool: {
+        filter: [
+          {
+            range: {
+              'artInitiation.dateInitiated': {
```
Comment (rcrichton) [inline packages/cdr/docker/jsreport/scripts/report-5f/content.js:17]: This field has changed in the latest index to be `artInitiation.date`. If you refresh you environment you will see this query doesn;t work unless you correct this field.

**Review by rcrichton — CHANGES_REQUESTED** [review]

File: packages/cdr/docker/jsreport/scripts/report-5f/content.js (hunk 2 above)
Comment (psmatsinhe) [reply packages/cdr/docker/jsreport/scripts/report-5f/content.js]: @rcrichton done

**Review by rcrichton — APPROVED** [review]

**Pull request merged**